app/Filament/Resources/NodeResource/Pages/CreateNode.php

```diff
@@ -212,6 +212,7 @@ public function form(Forms\Form $form): Forms\Form
                                     false => 'success',
                                 ]),
                             Forms\Components\ToggleButtons::make('public')
+                                ->default(config('panel.client_features.allocations.enabled', false))
```

Comment:
```suggestion
                                ->default(true)
```

This has nothing to do with "automatic allocations". `public` only controls if a node can be used for deployment.
"automatic allocations" is the feature that let users create a random allocation in the client area. (yes, the naming is a bit confusing)

Reply:
If you want you can change the label to something like `Useable for deployment`  to make it clearer and add a helper text that explains the field.